src/install.rs

```diff
@@ -320,35 +321,15 @@ async fn remove_package_from_environment(
     target_prefix: &Path,
     package: &PrefixRecord,
 ) -> miette::Result<()> {
-    // TODO: Take into account any clobbered files, they need to be restored.
-    // TODO: Can we also delete empty directories?
-
-    // Remove all entries
-    for paths in package.paths_data.paths.iter() {
-        match tokio::fs::remove_file(target_prefix.join(&paths.relative_path)).await {
-            Ok(_) => {}
-            Err(e) if e.kind() == ErrorKind::NotFound => {
-                // Simply ignore if the file is already gone.
-            }
-            Err(e) => {
-                return Err(e).into_diagnostic().wrap_err(format!(
-                    "failed to delete {}",
-                    paths.relative_path.display()
-                ))
-            }
+    match unlink_package(target_prefix, package).await {
```

Comment:
So this now takes care of all the todo's?

Reply:
Nope, one of the todo's just moved to rattler, the other one is taken care of :)